### PR TITLE
add embeddings for missing svs and new topics

### PR DESCRIPTION
**Pull request by lucy-kind**
verified embedding changes on local host nl

image image 
image
image
 image
image
Screenshot 2023-03-22 at 2 48 26 PM
Screenshot 2023-03-22 at 2 35 36 PM
Screenshot 2023-03-22 at 2 53 41 PM
image

### CODE REVIEW

**Review by jehangiramjad — APPROVED** [review]
Looks good to me. Thanks!

File: server/config/nl_page/chart_titles_by_sv.json [hunk 1]
```diff
@@ -1118,10 +1124,12 @@
   "dc/2wgh04kx6es88": "Median Income for Population Working in the Health Care and Social Assistance Industry",
   "dc/34t2kjrwbjd31": "Population Working in the Petroleum and Coal Products Manufacturing Industry",
   "dc/3ex11eg2q9hp6": "Median Income of Manufacturing Workers",
+  "dc/3jr0p7yjw06z9": "BLS Gambling Establishments",
```
Comment (pradh) [inline server/config/nl_page/chart_titles_by_sv.json:1127]: Just Gambling Establishments?

File: server/config/nl_page/chart_titles_by_sv.json [hunk 2]
```diff
@@ -1139,6 +1147,7 @@
   "dc/8lqwvg8m9x7z8": "Total Wages Earned by Population Working in Cattle Ranching and Farming",
   "dc/8p97n7l96lgg8": "Transportation and Warehousing Workers",
   "dc/8pxklrk2q6453": "Total Wages for Population Working in Retail Outside of a Store",
+  "dc/8xy9vzd30bqd": "USC Gambling Establishments",
```
Comment (pradh) [inline server/config/nl_page/chart_titles_by_sv.json:1150]: Just Gambling Establishments?

**Review by pradh — COMMENTED** [review]
Thanks!  Could you please include screenshots?

**Comment by lucy-kind** [comment]
Screenshot 2023-03-22 at 2 48 26 PM
Screenshot 2023-03-22 at 2 35 36 PM
Screenshot 2023-03-22 at 2 53 41 PM
m/63422135/227049750-1e06d5c8-97b3-4c4a-a7d9-93e15ab4d922.png">
image
image
image
image
image

**Comment by lucy-kind** [comment]
image image image image image

File: server/config/nl_page/chart_titles_by_sv.json (hunk 1 above)
Comment (lucy-kind) [reply server/config/nl_page/chart_titles_by_sv.json:1127]: Removed USC and changed names/descriptions for BLS SV

**Comment by lucy-kind** [comment]
Screenshots added in description!

**Comment by pradh** [comment]
Any idea if https://user-images.githubusercontent.com/63422135/227050444-370fb96a-0d22-4d3c-a092-364076369bf3.png improves to show the elementary rather than total after you fixed the strings?

File: server/config/nl_page/chart_titles_by_sv.json [hunk 3]
```diff
@@ -1118,10 +1124,12 @@
   "dc/2wgh04kx6es88": "Median Income for Population Working in the Health Care and Social Assistance Industry",
   "dc/34t2kjrwbjd31": "Population Working in the Petroleum and Coal Products Manufacturing Industry",
   "dc/3ex11eg2q9hp6": "Median Income of Manufacturing Workers",
+  "dc/3jr0p7yjw06z9": "BLS Gambling Establishments",
   "dc/3kwcvm428wpq4": "Rail Transportation Workers",
   "dc/4ky4sj05bw4nd": "Total Wages Earned by Population Working in the Air Transportation Industry",
   "dc/4mm2p1rxr5wz4": "Population Working in Miscellaneous Retail Stores",
   "dc/4z9xy6wn8xns1": "Median Income of Population Working in the Administrative and Support and Waste Management Services Industry",
+  "dc/5br285q68be6": "Population Working in Gambling Industries",
```
Comment (pradh) [inline server/config/nl_page/chart_titles_by_sv.json:1132]: This shows up as "Gambling Industries" and "Gambling Establishments".

Should we call this "Employees in Gambling Industry" (singular)?

**Comment by lucy-kind** [comment]
> 

It's still second. It makes sense though to have both of these in the top three: 
image

File: server/config/nl_page/chart_titles_by_sv.json (hunk 3 above)
Comment (lucy-kind) [reply server/config/nl_page/chart_titles_by_sv.json]: Done, updated and regenerated json

**Comment by pradh** [comment]
> > 
> 
> It's still second. It makes sense though to have both of these in the top three: image

The reason I think is the top one has "children in school", but the next one doesn't have "children in elementary school"

**Comment by pradh** [comment]
> > > 
> > 
> > 
> > It's still second. It makes sense though to have both of these in the top three: image
> 
> The reason I think is the top one has "children in school", but the next one doesn't have "children in elementary school"

Please add that in follow up if you don't want to regenerate stuff again.

**Review by pradh — APPROVED** [review]

**Comment by lucy-kind** [comment]
sg shanth, I added it to the alternatives in the embeddings sheet already so it will be in the next batch of upcoming changes!

**Comment by lucy-kind** [comment]
@pradh fixed the elementary school ranking and put new screen shot in description. thanks!

**Comment by pradh** [comment]
> @pradh fixed the elementary school ranking and put new screen shot in description. thanks!

LG thanks.

**Pull request merged**